webpack.config.js

```diff
@@ -488,14 +488,28 @@ async function initializeModuleFederation() {
             // Other containers will be forced to use the exact versions of shared modules that the web app provides.
             makeSingletonSharedModules([
                 'react',
+                'react-beautiful-dnd',
                 'react-bootstrap',
                 'react-dom',
                 'react-intl',
                 'react-redux',
                 'react-router-dom',
             ]),
         ],
-    }));
+    };
+
+    // Desktop specific code for remote module loading
+    moduleFederationPluginOptions.exposes = {
+        './root': 'components/root',
+        './crtWatcher': 'components/threading/channel_threads/posts_channel_reset_watcher',
+        './store': 'stores/redux_store.jsx',
+        './styles': 'sass/styles.scss',
+        './registry': 'registry',
+    };
+    moduleFederationPluginOptions.filename = 'remoteEntry.js';
```

Comment:
Does changing this not break the web app? This is currently output with a random name that is injected into root.html at build time.

Also, this should be named `remote_entry.js` based on our naming convention

Reply:
Nope, web app still runs without calling this module. I'll rename the file.